### PR TITLE
Ggantos

### PR DESCRIPTION
**Pull request by ggantos**

### CODE REVIEW

File: notebooks/microphysics.ipynb [hunk 1]
```diff
@@ -99,41 +59,97 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install ../../mlmicrophysics/."
+    "!pip install ../../mlmicrophysics"
```
Comment (djgagne) [inline notebooks/microphysics.ipynb]: Can you remove this line from the notebook? It is only valid on Casper or a local machine.

File: notebooks/microphysics.ipynb [hunk 2]
```diff
@@ -90,7 +50,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install numpy scipy pandas matplotlib xarray scikit-learn tensorflow netcdf4 h5netcdf tqdm pyyaml s3fs pyarrow"
+    "!pip install numpy scipy pandas matplotlib xarray scikit-learn tensorflow netcdf4 h5netcdf tqdm pyyaml s3fs pyarrow mlmicrophysics"
```
Comment (djgagne) [inline notebooks/microphysics.ipynb:53]: Can you add a second cell after this one that includes 
`! pip install --upgrade pandas`

**Review by djgagne — CHANGES_REQUESTED** [review]

File: notebooks/microphysics.ipynb (hunk 1 above)
Comment (ggantos) [reply notebooks/microphysics.ipynb]: done!

File: notebooks/microphysics.ipynb (hunk 2 above)
Comment (ggantos) [reply notebooks/microphysics.ipynb:53]: done!

**Review by djgagne — APPROVED** [review]

**Pull request merged**